src/email-helpers.js

```diff
@@ -39,6 +40,8 @@ module.exports = async function sendEmail(filename, url, sender, recipient, tran
     return;
   }
 
+  flag = branding;
```

Comment:
do you need this flag? or can you send branding directly to getmailOptions?

Reply:
no i can change it

Reply:
Please take a look at CI failures too